app/controllers/survey_controller.rb

```diff
@@ -2,4 +2,31 @@ class SurveyController < ApplicationController
   def index
     @questions = Question.order(:order)
   end
+  
+  def createQuestion
+    @question = Question.create(question: params[:question])
+    @question.order = @question.id # Did this because it is currently the convention in our seed data.
+    @question.save!
+    nextOrder = 0
```

Comment:
I made it dynamic in case a user inserted answers in a weird way (like if they put three answers in but left the field for answer 2 blank and instead had the answers in fields 1, 3, and 4).